### DISCUSSION TITLE
Multiple React version solutions

### DISCUSSION THREAD

**Discussion by vincenzopalazzo** [alephjs/aleph.js#231] [Q&A]
I ran the new version of the library in single-page mode and I receive the following error

```
Error: Invalid hook call. Hooks can only be called inside of the body of a function component. This could happen for one of the following reasons:
1. You might have mismatching versions of React and the renderer (such as React DOM)
2. You might be breaking the Rules of Hooks
3. You might have more than one copy of React in the same app
```

I user Material-UI as dependence and I think it pulls a different version of react, there any solution to fix it?

**Comment by c0per** [comment]
I'm stuck on this issue as well. Do you have any solutions?

**Reply by ije** [reply]
"aleph dev -r" may can help you
